examples/eli5/images/ELI5contrastive.svg

```diff
@@ -0,0 +1 @@
cap="butt" stroke-dasharray="4.0,3.0" d="m392.18896 60.249344l175.62207 220.28346" fill-rule="evenodd"/><path fill="#000000" fill-opacity="0.0" d="m392.18896 60.249344l175.62207 330.42517" fill-rule="evenodd"/><path stroke="#000000" stroke-width="1.0" stroke-linejoin="round" stroke-linecap="butt" stroke-dasharray="4.0,3.0" d="m392.18896 60.249344l175.62207 330.42517" fill-rule="evenodd"/><path fill="#000000" fill-opacity="0.0" d="m392.18896 170.27034l175.62207 -110.01575" fill-rule="evenodd"/><path stroke="#000000" stroke-width="1.0" stroke-linejoin="round" stroke-linecap="butt" stroke-dasharray="4.0,3.0" d="m392.18896 170.27034l175.62207 -110.01575" fill-rule="evenodd"/><path fill="#000000" fill-opacity="0.0" d="m392.18896 170.27034l175.62207 110.267715" fill-rule="evenodd"/><path stroke="#000000" stroke-width="1.0" stroke-linejoin="round" stroke-linecap="butt" stroke-dasharray="4.0,3.0" d="m392.18896 170.27034l175.62207 110.267715" fill-rule="evenodd"/><path fill="#000000" fill-opacity="0.0" d="m392.18896 170.27034l175.62207 220.40944" fill-rule="evenodd"/><path stroke="#000000" stroke-width="1.0" stroke-linejoin="round" stroke-linecap="butt" stroke-dasharray="4.0,3.0" d="m392.18896 170.27034l175.62207 220.40944" fill-rule="evenodd"/><path fill="#000000" fill-opacity="0.0" d="m392.18896 280.52887l175.62207 -220.28346" fill-rule="evenodd"/><path stroke="#000000" stroke-width="1.0" stroke-linejoin="round" stroke-linecap="butt" stroke-dasharray="4.0,3.0" d="m392.18896 280.52887l175.62207 -220.28346" fill-rule="evenodd"/><path fill="#000000" fill-opacity="0.0" d="m392.18896 280.52887l175.62207 -110.267715" fill-rule="evenodd"/><path stroke="#000000" stroke-width="1.0" stroke-linejoin="round" stroke-linecap="butt" stroke-dasharray="4.0,3.0" d="m392.18896 280.52887l175.62207 -110.267715" fill-rule="evenodd"/><path fill="#000000" fill-opacity="0.0" d="m392.18896 280.52887l175.62207 110.14172" fill-rule="evenodd"/><path stroke="#000000" stroke-width="1.0" stroke-linejoin="round" stroke-linecap="butt" stroke-dasharray="4.0,3.0" d="m392.18896 280.52887l175.62207 110.14172" fill-rule="evenodd"/><path fill="#000000" fill-opacity="0.0" d="m392.18896 390.66928l175.62207 -330.42517" fill-rule="evenodd"/><path stroke="#000000" stroke-width="1.0" stroke-linejoin="round" stroke-linecap="butt" stroke-dasharray="4.0,3.0" d="m392.18896 390.66928l175.62207 -330.42517" fill-rule="evenodd"/><path fill="#000000" fill-opacity="0.0" d="m392.18896 390.66928l175.62207 -220.40944" fill-rule="evenodd"/><path stroke="#000000" stroke-width="1.0" stroke-linejoin="round" stroke-linecap="butt" stroke-dasharray="4.0,3.0" d="m392.18896 390.66928l175.62207 -220.40944" fill-rule="evenodd"/><path fill="#000000" fill-opacity="0.0" d="m392.18896 390.66928l175.62207 -110.14172" fill-rule="evenodd"/><path stroke="#000000" stroke-width="1.0" stroke-linejoin="round" stroke-linecap="butt" stroke-dasharray="4.0,3.0" d="m392.18896 390.66928l175.62207 -110.14172" fill-rule="evenodd"/></g></svg>
```

Comment:
Don't add binary assets (GIFs, SVG, jpg files) to this repo, which we're trying to keep reasonably slim – we can find a better suited repo or CDN, ping me:)